spec/requests/sidebar_sections_controller_spec.rb

```diff
@@ -100,6 +101,20 @@
       expect(response.status).to eq(403)
     end
 
+    it "does not allow staff to create public section" do
```

Comment:
```suggestion
    it "does not allow moderator to create public section" do
```

Staff refers either moderators or admins.